packages/form/README.md

```diff
@@ -204,3 +203,37 @@ An demo using ReactJS can be found in our repository `examples`:
 
 - [StackBlitz playground](https://stackblitz.com/github/bholmesdev/simple-stack/tree/main/examples/form)
 - [GitHub](https://github.com/bholmesdev/simple-stack/tree/main/examples/form)
+
+## Sanitizing User Input
+
+To enhance the security of your application, it's crucial to sanitize user input to prevent Cross-Site Scripting (XSS) attacks. This process ensures that any potentially malicious scripts or code entered by users are neutralized before rendering on the webpage. Here, we'll guide you through using Zod `validation` in combination with the [sanitize-html](https://www.npmjs.com/package/sanitize-html) library for effective sanitization.
+
+### Install the sanitize-html library
+
+Before implementing sanitization, install the sanitize-html library along with its type definitions for TypeScript:
```

Comment:
```suggestion
```

We can remove the subheaders and jump straight to the installation instructions!